src/plugins/bubble/bubble-fr.hbs

```diff
@@ -0,0 +1,64 @@
+---
+{
+	"title": "Bubble - Fonctionalité provisoire",
+	"language": "fr",
+	"category": "Plugiciels",
+	"description": "Example pratique d'utilisation du chat wizard",
+	"tag": "bubble",
+	"parentdir": "bubble",
+	"altLangPrefix": "bubble",
+	"dateModified": "2020-10-21"
+}
+---
+{{>alert-provisional-demos}}
+
+<div class="wb-prettify all-pre"></div>
+<div lang="en">
+<p><strong>Need translation</strong></p>
+
+<p>Below is the code to achieve the demo on this page.</p>
+<pre><code>&lt;button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble trans-pulse"&gt;test&lt;/button&gt;
```

Comment:
Please remove the pulsing class named `trans-pulse`, since it should be added dynamically in Javascript.
```suggestion
<pre><code>&lt;button type="button" data-wb-doaction='{ "action": "openDialog", "source": "#mid-screen" }' class="wb-bubble"&gt;test&lt;/button&gt;
```